### PR TITLE
add more Subdomains to Rubicon-Project ruleset

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

File: src/chrome/content/rules/Rubicon-Project.xml [hunk 1]
```diff
@@ -15,53 +15,134 @@
...
+	<target host="beacon-us-sjc1.rubiconproject.com" />
+	<target host="beacon-us-west.rubiconproject.com" />
+	<target host="buyer.rubiconproject.com" />
+	<target host="chango.rubiconproject.com" />
+	<target host="connect.rubiconproject.com" />
+	<target host="connect-api.rubiconproject.com" />
+	<target host="ebsdrdmz.cs.rubiconproject.com" />
+	<target host="ebsproddmz.cs.rubiconproject.com" />
 	<target host="demand.rubiconproject.com" />
-	<target host="fastlane.rubiconproject.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Rubicon-Project.xml]: WFM

File: src/chrome/content/rules/Rubicon-Project.xml [hunk 2]
```diff
@@ -15,53 +15,134 @@
...
 
-	<!--	Direct rewrites:
-				-->
-	<target host="ads.rubiconproject.com" />
+	<target host="rubiconproject.com" />
+	<target host="www.rubiconproject.com" />
+	
+	<target host="49bc.rubiconproject.com" />
 	<target host="anvil.rubiconproject.com" />
-	<target host="arsenal.rubiconproject.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Rubicon-Project.xml]: WFM

File: src/chrome/content/rules/Rubicon-Project.xml [hunk 3]
```diff
@@ -15,53 +15,134 @@
...
+		- tap2-cdn.rubiconproject.com
 
 -->
 <ruleset name="Rubicon Project.com">
 
-	<!--	Direct rewrites:
-				-->
-	<target host="ads.rubiconproject.com" />
+	<target host="rubiconproject.com" />
+	<target host="www.rubiconproject.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Rubicon-Project.xml]: MCB

File: src/chrome/content/rules/Rubicon-Project.xml [hunk 4]
```diff
@@ -15,53 +15,134 @@
...
+	<target host="flapi1.rubiconproject.com" />
+	<target host="flds.rubiconproject.com" />
+	<target host="idm.rubiconproject.com" />
+	<target host="integration.rubiconproject.com" />
+	<target host="kb.rubiconproject.com" />
+	<target host="labs1.rubiconproject.com" />
 	<target host="login.rubiconproject.com" />
-	<target host="optimized-by.rubiconproject.com" />
-	<target host="pixel.rubiconproject.com" />
-	<target host="revv-static.rubiconproject.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Rubicon-Project.xml]: All three WFM.
You may find test urls with the help of search engines (e.g. `http://pixel.rubiconproject.com/tap.php`)

File: src/chrome/content/rules/Rubicon-Project.xml [hunk 5]
```diff
@@ -15,53 +15,134 @@
...
+	<target host="prebid-server.rubiconproject.com" />
 	<target host="revv.rubiconproject.com" />
 	<target host="s.update.rubiconproject.com" />
+	<target host="sas-api.rubiconproject.com" />
+	<target host="sdk.rubiconproject.com" />
+	<target host="secure-assets.rubiconproject.com" />
+	<target host="secured-by.rubiconproject.com" />
 	<target host="staged-by.rubiconproject.com" />
-	<target host="stats.aws.rubiconproject.com" />
-	<target host="tap.rubiconproject.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Rubicon-Project.xml]: WFM

File: src/chrome/content/rules/Rubicon-Project.xml (hunk 3 above)
Comment (numismatika) [reply src/chrome/content/rules/Rubicon-Project.xml]: maybe we should enable Upgrade Insecure Requests outside of the block http ressources mode

Comment (numismatika) [reply src/chrome/content/rules/Rubicon-Project.xml]: @J0WI can you check that again? The only thing i see browsing a little bit there is a failed geo lookup due to CORS from teletext.io, so obvious breakage.

**Pull request merged**